src/util/Constants.js

```diff
@@ -235,7 +235,6 @@ exports.Events = {
   GUILD_DELETE: 'guildDelete',
   GUILD_UPDATE: 'guildUpdate',
   GUILD_UNAVAILABLE: 'guildUnavailable',
-  GUILD_AVAILABLE: 'guildAvailable',
```

Comment:
Honestly, I still don't get why we removed this event, but keep `guildMemberAvailable` in our code. personally, this event should reappear again, to be able to know when a guild goes from unavailable to available. CC @iCrawl

Reply:
Me neither, but I think readding this would belong into a separate PR.
In similar manner could `guildMemberAvailable` be made more reliable, since we add members into the cache from all over the place, not just the two places were we currently emit that event.